modoboa_installer/utils.py

```diff
@@ -235,6 +235,16 @@ def success(message):
     printcolor(message, GREEN)
 
 
+def info(message):
+    """Print info message."""
+    printcolor(message, BLUE)
+
+
+def warning(message):
```

Comment:
While you're at it, could change evey instance of utils.printcolor(*,YELLOW/BLUE) to these functions?